resources/authgear/static/authgear.css

```diff
@@ -334,14 +330,14 @@ h1, h2, h3, h4, h5, h6 {
   border: none;
   border-radius: 0;
   border-bottom-style: solid;
-  border-bottom-width: 1px;
-  border-bottom-color: var(--color-text-shaded-2);
+  border-bottom-width: 2px;
```

Comment:
The color is correct now, but the border bottom should be kept in 1px.

<img width="734" alt="Screenshot 2021-02-02 at 11 07 09 AM" src="https://user-images.githubusercontent.com/388892/106534042-5484ba00-6547-11eb-8738-dcd19778e274.png">